dmutils/flask.py

```diff
@@ -45,7 +46,14 @@ def __init__(self, *args, compress_by_default: bool = False, **kwargs):
 
     def after_request(self, response):
         x_compression_safe = response.headers.pop("X-Compression-Safe", None)
-        compress = {"0": False, "1": True}.get(x_compression_safe, self.compress_by_default)
+        known_values = {"0": False, "1": True}
+        compress = known_values.get(x_compression_safe, self.compress_by_default)
+
+        if x_compression_safe not in known_values and x_compression_safe != "" and x_compression_safe is not None:
```

Comment:
This conditional could be simplified to

```suggestion
        if x_compression_safe and x_compression_safe not in known_values:
```

Reply:
Oh, also, I didn't think `not in` worked in Python? 🤔

Reply:
> This conditional could be simplified to

_Nearly_. Your line would not warn on actual python `False`, and I didn't go and check whether flask _always_ stringifies header values between view and middleware and don't really want to make the reader have to go & check up on that.